debian/changelog

```diff
@@ -77,7 +77,7 @@ ubuntu-advantage-tools (27.0.1) impish; urgency=medium
   * Add .gitignore and cleanup ignored directory .pytest_cache
   * apt-hook: mitigate failures with true
 
- -- Chad Smith <chad.smith@canonical.com>  Wed, 28 Apr 2021 13:55:28 -0600
```

Comment:
I think we should do this the other way around, no ? Otherwise we would have two changelog entries with almost the exact timestamp (27.0.1 - 27.0.2)

Reply:
I did it this way because the Friday date is what we have already released. So this PR just aligns `main` with what is actually installed in the wild. We could do it the other way but then we'll be retroactively editing a release date in the package, which feels weird but maybe its fine.

Do we remember what the actual release date was?

Reply:
Here's the diff to what's officially in the wild now for impish @xnox merged a quick fix for us direct into Impish, so we should pull that changelog entry into what we have into release-27 branch and or main too
```bash
snap install git-ubuntu
git-ubuntu clone  ubuntu-advantage-tools
cd ubuntu-advantage-tools
git checkout pkg/ubuntu/devel
git remote add upstream git@github.com:canonical/ubuntu-advantage-client.git
git fetch upstream
git diff upstream/release-27 debian/changelog
```

It get;s you this diff

```diff
diff --git a/debian/changelog b/debian/changelog
index 999d3907..6a56883b 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,4 +1,10 @@
-ubuntu-advantage-tools (27.1) impish; urgency=medium
+ubuntu-advantage-tools (27.1~21.10.1.1) impish; urgency=medium
+
+  * Cherrypick upstream pr #1681 to unbreak many migrations. LP: #1930741
+
+ -- Dimitri John Ledkov <xnox@ubuntu.com>  Wed, 16 Jun 2021 17:14:08 +0100
+
+ubuntu-advantage-tools (27.1~21.10.1) impish; urgency=medium
 
   * d/control:
     - specify debianutils min version
@@ -71,7 +77,7 @@ ubuntu-advantage-tools (27.0.1) impish; urgency=medium
   * Add .gitignore and cleanup ignored directory .pytest_cache
   * apt-hook: mitigate failures with true
 
- -- Chad Smith <chad.smith@canonical.com>  Fri, 07 May 2021 11:55:07 -0600
+ -- Chad Smith <chad.smith@canonical.com>  Wed, 28 Apr 2021 13:55:28 -0600
 
 ubuntu-advantage-tools (27.0) impish; urgency=medium
 
@@ -82,7 +88,7 @@ ubuntu-advantage-tools (27.0) impish; urgency=medium
       singular/plural pkgs
     - apt-hook: avoid reporting and counting duplicate package
       names (GH: #1578)
-    - fix: dont say reboot required when unnecessary (LP: #1926183)
+    - fix: don't say reboot required when unnecessary (LP: #1926183)
     - test: uncomment additional xenial upgrade tests
 
  -- Lucas Moura <lucas.moura@canonical.com>  Tue, 27 Apr 2021 15:31:06 -0300
@@ -466,9 +472,9 @@ ubuntu-advantage-tools (25.0~20.10.1~beta) groovy; urgency=medium
 ubuntu-advantage-tools (24.4) groovy; urgency=medium
 
   * New bug-fix-only release 24.4:
-     - uaclient.version bump to 24.4
-     - fips: honor additionalPackage directive from contract for bionic
-       (GH #1173)
+    - uaclient.version bump to 24.4
+    - fips: honor additionalPackage directive from contract for bionic
+      (GH #1173)
 
  -- Chad Smith <chad.smith@canonical.com>  Tue, 01 Sep 2020 11:14:39 -0600
 
```

I'd say we need to:
*  import Dmitri's changelog entry
* fix the date we have on 27.0.1 to be `Fri, 07 May 2021 11:58:03 -0600`
* pull in the indentation lintian changes for dont -> don't and intentation fixes.